app/src/main/java/org/mozilla/focus/settings/privacy/PrivacySecuritySettingsFragment.kt

```diff
@@ -193,11 +201,10 @@ class PrivacySecuritySettingsFragment :
     private fun updateStealthToggleAvailability() {
         val switch =
             preferenceScreen.findPreference(resources.getString(R.string.pref_key_secure)) as? SwitchPreferenceCompat
-        if (preferenceManager.sharedPreferences
-            .getBoolean(
-                    resources.getString(R.string.pref_key_biometric),
-                    false
-                )
+        if (preferenceManager.sharedPreferences.getBoolean(
```

Comment:
It would be preferable that the changes not related to this PR, especially indentation or code rearrangements are not included here.

Reply:
CI will not work if I revert the code